proposed/2021/NuGet.orgTFMs.md

```diff
@@ -0,0 +1,96 @@
+
+### Success: How do we know if we’ve solved this problem? 
+
+- By measuring the change in the percentage of successful installs and lowering the amount of [NU1202](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1202) errors in telemetry. 
+- By measuring the usage of target framework & supported platform filters on NuGet.org & Visual Studio. 
+- By gauging excitement/disappointment after blogging about this feature & analyzing sentiment on Twitter/GitHub/DevCom/etc. 
+
+### Audience: Who are we building for? 
+
+We are building this for the .NET developer who is browsing & making decisions about including a NuGet package into their project based on target framework & supported platform criteria. 
+
+## Explanation
+
+### Functional Explanation
+
+#### Context and Scope 
+When a developer accesses the details page for a NuGet package, they will see information on target frameworks in a dropdown menu. The framework information will be surfaced from the package. In cases where versions are not specified (such as .net) or other unexpected scenarios, ‘Version Not Specified’ or similar messages will be surfaced. 
+
+##### Minimal Requirements 
+- List of Frameworks on NuGet.org
+
+##### Non-Requirements 
+- List of Platforms on NuGet.org
+  - Users interviewed did not express need for supported platform information 
+  - Cannot be surfaced directly from package data and would require additional layer of soft calculation from engineering (no clear mapping scheme currently exists between target frameworks and supported platforms) 
+  - Can be revisited in future 
+- Filters for Frameworks on NuGet.org
+    - Users interviewed were ambivalent on filters, requires further investigation. Can be revisited in future 
+- Filters for Platforms on NuGet.org
+  - Users interviewed were ambivalent on filters, requires further investigation. Can be revisited in future 
+
+Although Target Frameworks & Supported Platforms are relatively similar in general contexts and often have overlap, we must clearly define each of these items based on their differences. Today we live in an ecosystem where Target Frameworks can represent both a supported platform, or a subset of supported platforms. 
+
+**Target Framework** – A target framework is a specification of the APIs available to an application or library. 
+
+Example: .NET Framework, .NET Core, .NET Standard, .NET 5 are target frameworks. 
+
+**Supported Platform** – A supported platform is a target that the APIs can be ran on. 
+
+Example: iOS, Android, Windows are platforms. 
+
+### Solution 
+
+- A .NET developer can view Frameworks of a package on NuGet.org: 
+![](../../meta/resources/NuGet.orgTFMs/PackageDetailsWithTFMs.png)
```

Comment:
Expanded by default? (dependencies are not)
Why isn't this the first section, even above dependencies?

Reply:
What order are the frameworks in? Lex order? Or some order we predefine?

Reply:
framework precedence would be a great ordering.

Reply:
How do we define framework precedence?

Reply:
https://nugettoolsdev.azurewebsites.net/5.8.0/framework-precedence 

There's a few APIs in NuGet.Frameworks IIRC.

Reply:
I think that framework precedence depends on project target framework. Or at least, with the current discussions about iOS/Mac TFMs in net6.0 it will be.

Reply:
Thanks @zivkan. So there may be a future-proofing concern here.
Just checking that we're not getting away from the intent of the spec a bit on this one?

Reply:
Naw, just that there's mostly a natural ordering we can still follow based on the net5.0 precedence defined today or shown similarly in the latest target framework docs!

Reply:
I like the notion of not expanded by default to allow for tracking as you mentioned elsewhere. I've added this, but it can always be removed if this isn't where we choose to land: `- We will have the list collapsed by default in order to track popularity.` 
I'd like it before Dependencies also, and will defer to @jcjiang on this.

Reply:
> Naw, just that there's mostly a natural ordering we can still follow based on the net5.0 precedence defined today or shown similarly in the latest target framework docs!
Fair enough. If there are APIs it should be easy enough to implement, and more importantly, it won't need maintenance..

Reply:
Great--included this change.